projects/1.21/assets/irregular-implements/irregular_implements/lang/zh_cn.json

```
@@ -0,0 +1,835 @@
+  "book.irregular_implements.guide.blocks.online_detector.page_1.title": "",
+  "book.irregular_implements.guide.blocks.pitcher_plant.description": "",
+  "book.irregular_implements.guide.blocks.pitcher_plant.name": "猪笼草",
+  "book.irregular_implements.guide.blocks.pitcher_plant.page_0.text": "[#](AA00AA)猪笼草[#]()是一种[#](00AAAA)可以充当无限水源[#]()的花。\\\n  \\\n它偶尔会用水[#](00AAAA)填充相邻的储罐[#]()！",
+  "book.irregular_implements.guide.blocks.pitcher_plant.page_0.title": "猪笼草",
+  "book.irregular_implements.guide.blocks.pitcher_plant.page_1.text": "可以在含有标签[#](00AAAA)#c:is_wet/overworld[#]()的生物群系中找到猪笼草，尤其是沼泽和丛林",
+  "book.irregular_implements.guide.blocks.pitcher_plant.page_1.title": "",
+  "book.irregular_implements.guide.blocks.quartz_glass.description": "",
+  "book.irregular_implements.guide.blocks.quartz_glass.name": "石英玻璃",
+  "book.irregular_implements.guide.blocks.quartz_glass.page_0.text": "[#](AA00AA)石英玻璃[#]()是一种特殊的玻璃，[#](00AAAA)玩家可以穿过[#]()，但对余下所有实体来说是实心方块。",
+  "book.irregular_implements.guide.blocks.quartz_glass.page_0.title": "石英玻璃",
+  "book.irregular_implements.guide.blocks.rain_shield.description": "",
+  "book.irregular_implements.guide.blocks.rain_shield.name": "驱雨盾",
+  "book.irregular_implements.guide.blocks.rain_shield.page_0.text": "[#](AA00AA)驱雨盾[#]()能够[#](00AAAA)在5个区块的范围内阻止降雨[#]()。\\\n  \\\n实际范围可以在服务器配置文件中更改。",
+  "book.irregular_implements.guide.blocks.rain_shield.page_0.title": "驱雨盾",
+  "book.irregular_implements.guide.blocks.rain_shield.page_1.text": "通入红石信号即可停用！",
+  "book.irregular_implements.guide.blocks.rain_shield.page_1.title": "",
+  "book.irregular_implements.guide.blocks.rainbow_lamp.description": "",
+  "book.irregular_implements.guide.blocks.rainbow_lamp.name": "彩虹灯",
+  "book.irregular_implements.guide.blocks.rainbow_lamp.page_0.text": "[#](AA00AA)彩虹灯[#]()在每种可能的红石信号强度（0至15）下都会呈现不同的颜色。",
+  "book.irregular_implements.guide.blocks.rainbow_lamp.page_0.title": "彩虹灯",
+  "book.irregular_implements.guide.blocks.shock_absorber.description": "",
+  "book.irregular_implements.guide.blocks.shock_absorber.name": "冲击吸收器",
+  "book.irregular_implements.guide.blocks.shock_absorber.page_0.text": "[#](AA00AA)冲击吸收器[#]()可以[#](00AAAA)完全吸收摔落伤害[#]()。\\\n  \\\n此外，摔落前的高度越高，红石信号的强度就越高。",
+  "book.irregular_implements.guide.blocks.shock_absorber.page_0.title": "冲击吸收器",
+  "book.irregular_implements.guide.blocks.sided_redstone.description": "",
+  "book.irregular_implements.guide.blocks.sided_redstone.name": "单面红石块",
+  "book.irregular_implements.guide.blocks.sided_redstone.page_0.text": "[#](AA00AA)单面红石块[#]()只会从其中的一个面输出红石信号。",
+  "book.irregular_implements.guide.blocks.sided_redstone.page_0.title": "单面红石块",
+  "book.irregular_implements.guide.blocks.spectre_lens.description": "",
+  "book.irregular_implements.guide.blocks.spectre_lens.name": "通灵棱镜",
+  "book.irregular_implements.guide.blocks.spectre_lens.page_0.text": "[#](AA00AA)通灵棱镜[#]()能够被放置在信标上，[#](00AAAA)这样便能无视距离，始终保持信标效果。[#]() \\\n  \\\n尽管如此，玩家仍然需要与棱镜[#](FF5555)在同一个维度[#]()，且棱镜所在位置必须被加载。",

```

```suggestion
  "book.irregular_implements.guide.blocks.spectre_lens.page_0.text": "[#](AA00AA)通灵棱镜[#]()能够被放置在信标上，[#](00AAAA)这样便能无视距离，始终为你施加信标效果。[#]() \\\n  \\\n尽管如此，玩家仍然需要与棱镜[#](FF5555)在同一个维度[#]()，且棱镜所在位置必须被加载。",
```
另外感觉根据这个功能，Lens可能应该翻译为`透镜`